_guides/software-request-guide.md

```diff
@@ -0,0 +1,89 @@
+* Determination of whether or not the requester believes that the piloted technology is acceptable and should be a candidate for full approval;
+* Path to attain full security, and Section 508 approval; and
+* Evidence of all costs have been identified and budgeted within the requesting office and as required by other affected offices, to include out-year operations and maintenance.
+
+### Monitor
+After your software request is entered in ServiceNow, you should receive an email confirmation. Be sure to read that email since it will provide your service ticket number.  Any follow-up communication will also be sent to you in email from the IT Standards Desk. Keep an eye out for requests for more information to be added to your ticket. If you do not receive an email with your request ticket information, you can still review this information in ServiceNow by viewing your open tickets. For any questions, contact the IT Standards Team at it-standards@gsa.gov.
+
+### Approve
+It can typically take up to 2 weeks for a request to be completed for Standard software. After your software request is approved, the ticket is marked at closed and you'll be contacted via phone or email from your local IT Service Desk. Remember, you can always check your ticket status in [ServiceNow](https://gsa.service-now.com/GSA_Self-Service/home.do).
+
+### Acquire
+Once the software is approved, you will work with your local Contract Officer (CO) to purchase a software license. Reach out to your supervisor to see who the correct CO would be.
+
+### Use
+If you requested web-based software, then you may go ahead and start using it.
+All other types of software will be installed by your local IT Service Desk within 1-2 business days of your request being approved. Contact the IT Service Desk at 1-866-450-5250 or itservicedesk@gsa.gov for questions.
+
+### Discontinue
+All software is assigned an expiration date. This date is when the software will be re-evaluated for GSA use. If software is discontinued, you will be notified via email.
+When you no longer need the software, please enter a ticket in ServiceNow to have it removed from your equipment.
+<br>
+<br>
+Resources: [ServiceNow](#)
+
+### FAQ's
+
+<ul class="usa-accordion">
+  <li>
+    <button class="usa-accordion-button"
+      aria-expanded="true" aria-controls="Question-1">
+      What is the difference between Standard and Non-standard software?
+    </button>
+    <div id="Question-1" class="usa-accordion-content">
+      <p>
+      Standard software has previously been evaluated and approved to be included in the IT Standards List in GEAR. You must still put in a request for Standard software. This type of request typically takes 1-2 weeks to complete.
+      Non-standard software has not been evaluated for inclusion in the IT Standards List and must complete a full review for Security, 508 Accessibility, and Enterprise Architecture requirements. This review and evaluation process can take 4-6 weeks to complete.
```

Comment:
I'd split the estimates out to their own FAQ question, or move them up and make them more prominent within the description/diagram above.